examples/tutorials/Druggablity_assessment_with_Fpocket.ipynb

```diff
@@ -0,0 +1,2112 @@
+{
```

Comment:
<p>Small suggestion to make it more clearer and reader-friendly, how about changing it to "Understanding Druggability".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>